webapp/src/main/java/rocks/metaldetector/service/auth/TokenPair.java

```diff
@@ -0,0 +1,9 @@
+package rocks.metaldetector.service.auth;
+
+import org.springframework.http.ResponseCookie;
+
+public record TokenPair(
+    String accessToken,
+    ResponseCookie refreshToken
+) {
+}
```

Comment:
```suggestion
) {}
```
Like in AccessTokenResponse